src/backend/Csrs.Api/ApiGateway/ApiGatewayHandler.cs

```diff
@@ -37,6 +39,7 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
                                     Encoding.UTF8,
                                     "application/json");//CONTENT-TYPE header
 
+                _logger.LogInformation("The _apiGatewayOptions.BasePath is ", path);
```

Comment:
It is no longer base path but the absolute path i guess. So from hat standpoint the logging is little confusing. Let's clean it next time after tetsing